app/controllers/api/v1/accounts_controller.rb

```diff
@@ -85,7 +85,7 @@ def unblock
   def relationships
     ids = params[:id].is_a?(Enumerable) ? params[:id].map(&:to_i) : [params[:id].to_i]
 
-    @accounts    = Account.where(id: ids).select('id')
+    @accounts    = Account.where(id: ids).select('id, domain')
```

Comment:
i think i was testing something that i didn't quite understand from @hach-que 's closed patch, and forgot to revert it